src/ray/core_worker/core_worker.cc

```diff
@@ -384,7 +383,21 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
             "CoreWorker.HandleException");
       }));
 
-  experimental_mutable_object_manager_.reset(new ExperimentalMutableObjectManager());
+  mutable_object_manager_ = std::make_shared<MutableObjectManager>();
+
+  auto raylet_channel_client_factory = [this](const NodeID &node_id) {
+    const rpc::GcsNodeInfo *node_info = gcs_client_->Nodes().Get(node_id);
+    RAY_CHECK(node_info) << "No GCS info for node " << node_id;
+    std::shared_ptr<rpc::NodeManagerWorkerClient> grpc_client =
+        rpc::NodeManagerWorkerClient::make(
+            node_info->node_manager_address(),
+            node_info->node_manager_port(),
+            *mutable_obj_network_manager_->client_call_manager());
+    return std::make_shared<raylet::RayletClient>(std::move(grpc_client));
+  };
+  mutable_obj_network_manager_ =
+      std::make_shared<MutableObjectNetworkManager<CoreWorkerPlasmaStoreProvider>>(
+          plasma_store_provider_, raylet_channel_client_factory);
```

Comment:
Discussed offline: Might be cleaner to introduce a new "MutableObjectProvider", since we don't really need the  PlasmaStoreProvider's functionality anyway.

Reply:
Done.